roles/horizon/templates/etc/openstack-dashboard/local_settings.py

```diff
@@ -363,7 +363,7 @@
 }
 
 # Enables upload from remote location
-IMAGES_ALLOW_LOCATION = "{{ horizon.glance.allow_location }}"
+IMAGES_ALLOW_LOCATION = {{ horizon.glance.allow_location | default('false') | bool }}
```

Comment:
this is only needed because when `horizon.glance.allow_location` was added it didn't get an entry in `roles/horizon/defaults/main.yml` we should fix that instead.   Usually we only use defaults like this when we're grabbing a value from another role that may or may not be available to the current role.